test/login.js

```diff
@@ -142,4 +158,59 @@ describe('Login', function() {
     });
   });
 
+  it('should log a whitelisted user', function(done) {
+    var user = apos.users.newInstance();
+
+    user.firstName = 'Admin';
+    user.lastName = 'Test';
+    user.title = 'Admin Test';
+    user.username = 'admin-test';
+    user.password = 'crookshanks';
+    user.email = 'admintest@aol.com';
+    user.lastLogin = new Date();
+    user.groupIds = [ apos.users.options.groups[1]._id ]; // admin group
+
+    apos.users.insert(apos.tasks.getReq(), user, function(err) {
+      assert(!err);
+      return request.post('http://localhost:7901/login', {
+        form: { username: 'admin-test', password: 'crookshanks' },
+        followAllRedirects: true,
+        jar: loginLogoutJar
+      }, function(err, response, body) {
+        assert(!err);
+        assert.equal(response.statusCode, 200);
+        assert(body.match(/logout/));
+        return done();
+      });
+    });
+  });
+
+  it('should disable an inactive user', function(done) {
+    var user = apos.users.newInstance();
+
+    user.firstName = 'Random';
+    user.lastName = 'Test';
+    user.title = 'Random Test';
+    user.username = 'random-test';
+    user.password = 'crookshanks';
+    user.email = 'randomtest@aol.com';
+    user.lastLogin = new Date();
```

Comment:
If their last login was just a moment ago, shouldn't they be able to get in again (they are not not inactive)?

Reply:
In this test, I configured the parameter `inactivityDuration` to 0 in purpose. This way, we check the user is really disabled after a login connection if the inactivity duration is reached.

Reply:
Did you also include a test that it doesn't incorrectly time out users who have not timed out?

Reply:
If so then the 0 approach for this test of a positive case is OK.

Reply:
Yes. As I configured the `apos` object with a `inactivityDuration` with 0, every existing "logging" tests in this file check if non-timed out users can log in (line 99 and line 115).

And I also check if a user belonging to `neverDisabledGroups` ("admin" in this file), even timed-out, can still log in (line 161)

Reply:
I added the missing test